tests/datasets/test_geo.py

```diff
@@ -156,20 +160,29 @@ def test_and_nongeo(self, dataset: GeoDataset) -> None:
 
 
 class TestRasterDataset:
-    @pytest.fixture(params=[True, False])
+    @pytest.fixture(params=zip([True, False], [["R", "G", "B"], None]))
     def naip(self, request: SubRequest) -> NAIP:
         root = os.path.join("tests", "data", "naip")
+        bands = request.param[1]
         crs = CRS.from_epsg(3005)
         transforms = nn.Identity()
-        cache = request.param
-        return NAIP(root, crs=crs, transforms=transforms, cache=cache)
-
-    @pytest.fixture(params=[True, False])
+        cache = request.param[0]
```

Comment:
Minor: I would change the order of these request params and the order in which these variables are defined to match the order of arguments in the next line